iib/workers/s3_utils.py

```diff
@@ -0,0 +1,38 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+import logging
+
+import boto3
+
+from iib.exceptions import IIBError
+from iib.workers.config import get_worker_config
+
+log = logging.getLogger(__name__)
+
+
+def upload_file_to_s3_bucket(file_path, key_prefix, file_key):
+    """
+    Upload artifact file to AWS S3 bucket.
+
+    :param str file_path: the path of the file locally where the artifact file is stored
+    :param str key_prefix: the logical location of the file in the S3 bucket
+    :param str file_key: the name of the file in S3 bucket
```

Comment:
since it `file_key` should be name, what about naming this parameter differently?
What about `s3_file_name`? And maybe update second parameter this in similar way `s3_key_prefix` or just `s3_prefix`?

Why I am asking to think about it is the fact that I had to read what `file_key` is to understand that it is just file name on s3.